src/CMakeLists.txt

```diff
@@ -91,6 +91,11 @@ if (USE_TORCH)
     backends/torch/db_lmdb.cpp
     backends/torch/native/templates/nbeats.cc
     backends/torch/native/templates/vit.cc
+    backends/torch/native/templates/ttransformer.cc
+    backends/torch/native/templates/tt/tembedder.cc
```

Comment:
"Pedantic": can we use `ttransformer/` as the directory name as a good practice using the template name directly ?

Reply:
ok